views/templates/admin/configure.tpl

```diff
@@ -23,6 +23,7 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
+{$hookConfigureTop}
```

Comment:
Hello !

I understand that you are trying to implement a way for modules (probably https://github.com/prestashop/ps_mbo) to inject content at the top and bottom of this template.

But I think maybe we dont even need a hook to do this 🤔  .

If I am not wrong, the aim is to be able to inject a Addons-related banner in the bottom, and this must be done in the corporate distribution? The corporate distribution will be a full PrestaShop ZIP with modules and theme embedded.

Themes have the ability to skin existing module templates. So the them cant extend and/or replace any template from the module. So if I am not wrong, we already have all the means necessary to inject content at the bottom and top of a module template when we use the theme.

If we wanted to inject content in the middle, we would need a hook or a Twig block at the right place. But for adding content at the bottom or at the top, I think theme already have all the capabilities they need, so I don't think these hooks are useful.

Would be happy to be know what @kpodemski or @Hlavtox would say as they probably have implemented this many times before.

Reply:
Unfortunately, me, friends who make and sell modules, my clients, anybody never needed any hook like this. :-)

I don't know the purpose of this. Only thing I can think of is maybe displaying some information from blog module on sitemap module configuration?

Reply:
@Hlavtox yes, totally, this hook, IMO, should be removed from this module and moved into the core 👍